data_steward/analytics/cdr_ops/ehr_snapshot_qc.py

```diff
@@ -0,0 +1,59 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.3.0
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# + tags=["parameters"]
+project_id = ""
+old_ehr = ""
+new_ehr = ""
+# -
+
+# # QC for EHR Snapshot
+#
+# Quality checks performed on a new EHR Snapshot dataset and comparison with ehr_ops dataset.
+
+# +
+import pandas as pd
+
+pd.options.display.max_rows = 120
+
+# -
+
+# # Table comparison
+# The snapshot should contain the same tables as ehr_ops.
+
+query = f'''
+SELECT table_id
+FROM `{project_id}.{old_ehr}.__TABLES__`
+WHERE table_id LIKE '%person%'
+AND NOT (table_id LIKE '%mapping%' OR table_id LIKE '%unioned%')
+EXCEPT DISTINCT
+SELECT table_id
+FROM `{project_id}.{new_ehr}.__TABLES__`
+WHERE table_id LIKE '%person%'
+AND NOT (table_id LIKE '%mapping%' OR table_id LIKE '%unioned%')
+'''
+pd.read_gbq(query, dialect='standard')
```

Comment:
Feel free to relocate `execute()` to some common module if importing it from `combined` has a bad smell; just pls refactor `combined` usages if you do.

Reply:
Thanks, I used the rdr notebook as reference and did not know of this.

Reply:
@mark-velez Thanks, I wanted to pass a different client but didn't want to change the combined notebook. I can parametrize client and place it in `cdr_ops/notebook_utils.py` or `utils/notebook.py`.

Reply:
I like the idea of placing `execute` in a util so it can be imported as needed.  We could do the same with the `CLIENT` object.  Define it in the util and import as needed.  This would allow a dev to create one or just import the one from the utility if nothing extra is needed.

Thoughts @mark-velez and @nishanthpp93 .

Reply:
@lrwb-aou just to be sure, the `project_id` for that shared client object would be set to the default inferred from the environment, right? I think that would be convenient, but the prereqs should be specially documented. If the environment does **not** have application default credentials set up then would the first import of the util package raise `DefaultCredentialsError` as described [here](https://google-auth.readthedocs.io/en/latest/reference/google.auth.html#google.auth.default)? :thinking: